Talend/ui
# chore: upgrade webpack 5

**What is the problem this PR is trying to solve?**

* we use the outdated webpack 4
* it brings security issue:
webpack4 => watchpack => ... => glob-parent
* lack of features (web worker, module federation)

**What is the chosen solution to this problem?**

* Upgrade to webpack 5
* Upgrade all loader

QA validations tasks:
* yarn list --pattern webpack is as flat as possible ✅ 
* yarn list --pattern loader is as flat as possible ✅
* yarn postinstall ✅ (icons, tokens, theme are OK)
* yarn run pre-release ✅ (check dependencies.json are generated along report and map files)

**Please check if the PR fulfills these requirements**

- [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [x] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR



**[ ] This PR introduces a breaking change**
